src/Amp/Database/Datasource.php

```diff
@@ -175,6 +175,16 @@ function updateHost() {
     }
   }
 
+  /**
+   * Convert a PDO driver name to a PHP extension name.
+   */
+  function convertPDODrivertoPHPExtension($driver) {
+    if ($driver == 'mysql') {
```

Comment:
@mollux , how about making this more conditional, e.g. `if (!extension_loaded('mysql') && $driver == 'mysql')`

This way, we can have one build of `amp.phar` which (a) works interchangeably on older versions of PHP and (b) also works on PHP7? Providing that compat should simplify the sequencing of merging/deploying changes among the various projects.